dissect/target/plugins/os/windows/adpolicy.py

```diff
@@ -69,16 +69,16 @@ def _xmltasks(self, policy_dir):
                 task_file_stat = task_file.stat()
                 xml = task_file.read_text()
                 tree = ElementTree.fromstring(xml)
-                for task in tree.findall("Task"):
-                    properties = task.find("Properties")
+                for task in tree.findall(".//{}Task"):
```

Comment:
I don't know much about XPath, but I couldn't find what the `{}` does in this context.

Reply:
It's to access nodes with empty namespaces. {*} would be better yet maybe.

Reply:
For some reason the xml contained empty namespaced nodes. {*} will allow the query to access all namespaced nodes.